pkgs/tools/typesetting/tex/tex-match/default.nix

```
@@ -0,0 +1,31 @@
+{ rustPlatform, fetchFromGitHub, gtk3, pkg-config, glib, lib }:
+
+let
+  version = "1.2.0";
+in
+rustPlatform.buildRustPackage {
+  pname = "tex-match";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "zoeyfyi";
+    repo = "TeX-Match";
+    rev = "v${version}";
+    sha256 = "093wrw4pq1llaivc644r93g96wq5alvh93rdswr6p58zqkqw4zgg";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ pkg-config glib ];
+
+  buildInputs = [ gtk3 ];
+
+  cargoSha256 = "1ci139q3dzbgsq0pk2hr351v0h0ql2psvqr92iap63apx3v6kqir";
```

(After fixing `sha256`)
```
hash mismatch in fixed-output derivation '/nix/store/cszyrkxl3ckn44c9a0dgq917fqwxng3z-tex-match-1.2.0-vendor.tar.gz':
  wanted: sha256:1ci139q3dzbgsq0pk2hr351v0h0ql2psvqr92iap63apx3v6kqir
  got:    sha256:1sm2fd3dhs59rvmfjzrfz0qwqzyc9dllb8ph0wc2x0r3px16c71x
```
```suggestion
  cargoSha256 = "1sm2fd3dhs59rvmfjzrfz0qwqzyc9dllb8ph0wc2x0r3px16c71x";
```